### PR TITLE
Fix the installation paths

### PR DESCRIPTION
**Pull request by seisman**
Related to #108.

### CODE REVIEW

File: source/utilities/distaz.rst [hunk 1]
```diff
@@ -16,7 +16,12 @@ distaz
 
 仅以 C 语言版本的 ``distaz`` 为例，用如下命令即可编译::
 
-   cc distaz.c -o distaz -lm
+    $ cc distaz.c -o distaz -lm
+
+并将生成的可执行文件移动到 :file:`~/bin/` 目录下::
+
+    $ mkdir ~/bin/
+    $ mv distaz ~/bin/
```
Comment (core-man) [inline source/utilities/distaz.rst:25]:
```suggestion

若 :file:`~/bin/` 路径尚未添加到 PATH 中，则执行如下命令::

     $ echo 'export PATH=${HOME}/bin:${PATH}' >> ~/.bashrc
     $ source ~/.bashrc

```

**Comment by core-man** [comment]
Do we need to make a note for GMT and SAC on the metadata page so that the reader can install them in `~/opt/`.

**Comment by seisman** [comment]
> Do we need to make a note for GMT and SAC on the metadata page so that the reader can install them in `~/opt/`.

Not a bad idea, but perhaps after we finish https://github.com/seismo-learn/seismology101/pull/41?

**Comment by core-man** [comment]
> > Do we need to make a note for GMT and SAC on the metadata page so that the reader can install them in `~/opt/`.
> 
> Not a bad idea, but perhaps after we finish [seismo-learn/seismology101#41](https://github.com/seismo-learn/seismology101/pull/41)?

Okay~

File: source/utilities/distaz.rst [hunk 2]
```diff
@@ -16,7 +16,17 @@ distaz
 
 仅以 C 语言版本的 ``distaz`` 为例，用如下命令即可编译::
 
-   cc distaz.c -o distaz -lm
+    $ cc distaz.c -o distaz -lm
+
+并将生成的可执行文件移动到 :file:`~/bin/` 目录下::
+
+    $ mkdir ~/bin/
```
Comment (core-man) [inline source/utilities/distaz.rst:23]:
```suggestion
    $ mkdir -p ~/bin/
```

File: source/cps/install.rst [hunk 3]
```diff
@@ -13,9 +13,9 @@
 
         $ tar -xvf NP330.Oct-29-2020.tgz
 
-    将解压得到的 :file:`PROGRAMS.330` 文件夹移至 ``src`` 目录下
-    （若尚未建立 ``src`` 目录，可在终端键入 ``mkdir ~/src`` 新建此目录）::
+    将解压得到的 :file:`PROGRAMS.330` 文件夹移至 :file:`~/src/` 目录下::
```
Comment (core-man) [inline source/cps/install.rst:16]:
```suggestion
    将解压得到的 :file:`PROGRAMS.330` 文件夹移至 :file:`~/src/` 目录下，并重命名为 :file:`CPS` ::
```

File: source/cps/install.rst [hunk 4]
```diff
@@ -13,9 +13,9 @@
...
 
+        $ mkdir -p ~/src/
         $ mv PROGRAMS.330 ~/src/CPS
 
     读者也可以根据自己的习惯，将 :file:`PROGRAMS.330` 放在其他目录下。
```
Comment (core-man) [inline source/cps/install.rst:21]:
```suggestion
```

**Review by core-man — COMMENTED** [review]
CPS 的 bin 路径添加到 PATH 中需要更新新的路径

**Comment by seisman** [comment]
> CPS 的 bin 路径添加到 PATH 中需要更新新的路径

Done.

**Review by core-man — APPROVED** [review]

**Pull request merged**